### PR TITLE
Remove Hardcoded Fees

### PR DESCRIPTION
**Pull request by alistair-singh**
Resolves: SNO-120

Pulls the fees from on chain data.

- [x] Complete the logic for ERC20 Dot fees.
- [ ] ~~Big Integer Math for conversions.~~ Does not support the decimal points needed.
- [x] Use Numbers instead of strings to check whether the user has enough balance to pay fees.
- [ ] ~~Deploy the latest IncentivizedOutboundChannel contract to staging and test.~~ The latest version is already deployed to staging.

### CODE REVIEW

File: src/redux/actions/bridge.ts [hunk 1]
```diff
@@ -151,8 +154,47 @@ export const updateBalances = ():
...
+    } = getState() as RootState;
+
+    const denomination = 10 ** 18;
+
+    try {
+      switch (swapDirection) {
+        case SwapDirection.EthereumToPolkadot:
+          // TODO: Proper handling of conversion
+          const erc20Dot: any = await incentivizedOutboundChannelContract!.methods.fee().call();
+          const erc20DotFormatted = (Number(erc20Dot) / denomination).toString();
```
Comment (vgeddes) [inline src/redux/actions/bridge.ts:182]: Problem here is that `erc20Dot` can be 256-bits (when converted from a string to BigNumber). However the `Number` type is only 64-bits, so there is a chance of truncation.

@musnit @wildmolasses @sumitsnk Any of you happen to know any ethereum-ecosystem JS libs for currency formatting?

Comment (musnit) [reply src/redux/actions/bridge.ts:182]: we should probably use ethers.utils.formatUnits > https://docs.ethers.io/v5/api/utils/display-logic/#unit-conversion

Comment (musnit) [reply src/redux/actions/bridge.ts:182]: And yeh should we be using Number or Balance from polkadotjs? irrespective, we do need a way to convert between BigNumber which we should use for ethereum assets and Number/Balance for polkadot assets with an explicit decision on how to handle truncation (eg: accept it and show it to the user, disallow it/block big numbers and show an error, etc)

Comment (alistair-singh) [reply src/redux/actions/bridge.ts]: I have used the number and formatting libraries native to each chain.

**Review by vgeddes — APPROVED** [review]
Looks good! 🚀

**Pull request merged**